spec/musicalbum_spec.rb

```diff
@@ -0,0 +1,28 @@
+require_relative '../item_content/musicalbum'
+
+RSpec.describe MusicAlbum do
+  describe '#initialize' do
+    it 'initializes a music album with on_spotify attribute' do
+      music_album = MusicAlbum.new(true)
+      expect(music_album.genre).to eq('Music')
+      expect(music_album.author).to eq('Unknown')
+      expect(music_album.source).to eq('Unknown')
+      expect(music_album.label).to eq('Unknown')
+      expect(music_album.on_spotify).to be true
+    end
+  end
+
+  describe '#move_to_archive' do
+    it 'moves a music album to the archive if it can be archived' do
+      music_album = MusicAlbum.new(true)
+      music_album.move_to_archive
+      expect(music_album.instance_variable_get(:@archived)).to be true
+    end
+
+    it 'does not move a music album to the archive if it cannot be archived' do
```

Comment:
- You have done great building out your capstone project :+1: :+1: But there are a few corrections and requirements you need to meet before moving to the next milestone. :+1: 

 - [ ] Looking at the video below, you can see that you have some tests failing, About 4 of the test cases are failing, I'll suggest you look into this and fix them. :+1: :+1: 

- - Video reference:
 

https://github.com/manzitresor/Catalog-of-my-things/assets/107632780/94182465-7b43-4f8b-9d66-9fee43c5f2f6